layers/+lang/c-c++/packages.el

```diff
@@ -174,3 +175,7 @@
     :post-init
     (dolist (mode '(c-mode c++-mode))
       (spacemacs/setup-helm-cscope mode))))
+
+(defun c-c++/post-init-counsel-gtags ()
+  (spacemacs/counsel-gtags-define-keys-for-mode 'c-mode)
+  (spacemacs/counsel-gtags-define-keys-for-mode 'c++-mode))
```

Comment:
Is there any reason why you have done this only for C/C++ and not for all the other modes where `spacemacs/helm-gtags-define-keys-for-mode` is used currently?